CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
 * Fix `Lint/FormatParameterMismatch` for splatted last argument. ([@zverok][])
 * [#3853](https://github.com/bbatsov/rubocop/pull/3853): Fix false positive in `RedundantParentheses` cop with multiple expression. ([@pocke][])
 * [#3870](https://github.com/bbatsov/rubocop/pull/3870): Avoid crash in `Rails/HttpPositionalArguments`. ([@pocke][])
+* [#3879](https://github.com/bbatsov/rubocop/pull/3879): Properly handle emacs and vim magic comments for `FrozenStringLiteralComment`. ([@backus][])
```

Comment:
I just wrote one comment and my internet died. :-) I'll come back to this PR a bit later - had a long trip from Europe to Thailand earlier today.